WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -0,0 +1,75 @@
+<table class="table table-striped table-condensed admin-competitions">
+  <% toggle_tbody_message = title.include?("Recent") ? "hide" : "show" %>
+  <thead>
+    <tr class="admin-table-header">
+      <th>
+        <%= "#{title} (#{competitions.count})" %>
+      </th>
+      <th class="admin-delegate">
+        Delegate(s)
+      </th>
+      <th class="admin-date">
+        Date
+      </th>
+      <th class="admin-date">
+        Announced
+      </th>
+      <th class="admin-date">
+        Report posted
+      </th>
+      <th class="admin-date">
+        Results posted
+      </th>
+      <th class="admin-button">
+        <span class="toggle-body">(<%= toggle_tbody_message %>)</span>
+      </th>
+    </tr>
+  </thead>
+  <tbody class="<%= "hidden" if toggle_tbody_message == "show" %>">
+    <% competitions.each_with_index do |competition, index| %>
+      <% announced = competition.announced_at %>
+      <% report = competition.delegate_report.posted_at %>
+      <% results = competition.results_posted_at %>
+      <tr>
+        <td>
+          <i class="flag f16 <%= competition.country.iso2.downcase %>"></i><%= link_to competition.cellName, competition_path(competition) %>
+          <br /><strong><%= competition.country_name %></strong>, <%= competition.cityName %>
+        </td>
+        <td class="admin-delegate">
+          <%= users_to_sentence competition.delegates %>
+        </td>
+        <td class="admin-date">
+          <%= wca_date_range(competition.start_date, competition.end_date) %>
+        </td>
+        <% if announced
+            days_announced = (competition.start_date - announced.to_date).to_i
+            announced_class = announced_class(days_announced)
+            announced_content = "#{announced.strftime("%b %d, %Y")} (#{days_announced})"
```

Comment:
We have date formats declared here: https://github.com/cubing/worldcubeassociation.org/blob/d7ded260b4e029fba58d1255f37a6f2be04411cc/WcaOnRails/config/locales/rails.en.yml#L34-L37. We don't want more copies of them. Use one of our existing formats by doing this:

```
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> bin/rails c
Loading development environment (Rails 4.2.1)
irb(main):001:0> I18n.default_locale
=> :en
irb(main):002:0> I18n.l Time.now, format: :short
=> "31 Aug 11:25"
irb(main):003:0> I18n.l Time.now, format: :default
=> "Wed, 31 Aug 2016 11:26:42 -0700"
irb(main):004:0> I18n.l Time.now, format: :long
=> "August 31, 2016 11:26"
irb(main):005:0> 
```